submissions/serializers.py

```diff
@@ -90,6 +90,21 @@ class ScoreSerializer(serializers.ModelSerializer):
     # Ensure that the created_at datetime is not converted to a string.
     created_at = DateTimeField(format=None, required=False)
 
+    annotations = serializers.SerializerMethodField()
+    def get_annotations(self, obj):
+        """
+        Inspect ScoreAnnotations to attach the annotation_type of any found annotations.
+        """
+        annotations = ScoreAnnotation.objects.filter(score_id=obj.id)
+        return [
```

Comment:
Adding the companion `ScoreAnnotationSerializer` was the closest I could get here, is that similar to what you were thinking?